imap/message.c

```diff
@@ -1964,8 +1964,7 @@ static void message_parse_content(struct msg *msg, struct body *body,
         len = endline - line;
         msg->offset += len;
 
-        if (line[0] == '-' && line[1] == '-' &&
-            message_pendingboundary(line, len, boundaries)) {
+        if (message_pendingboundary(line, len, boundaries)) {
```

Comment:
This does mean we'll call this method for every single line in an email when before we wouldn't. Performance wise this maybe doesn't matter since messages don't have too many lines. What do you think?

Reply:
Switched back to the original test but checking length first